client/app/_users/users.scss

```diff
@@ -0,0 +1,20 @@
+.users-table {
+    width: 100%;
+    table-layout: fixed;
+
+    &-row {
+        &._name-wrapper {
+            flex: auto;
+            width: auto;
+            overflow: hidden;
+            text-overflow: ellipsis;
```

Comment:
Does it work w/o `max-width`? and do you need this here at all?

Reply:
fixed